index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Microverse!</title>
+</head>
+<body>
+    <header>
+        <div>
+            <img src="src/img/logo.png" class="logo" alt="Portfolio">
+            <img src="src/img/menu-button.png" class="menu-button" alt="Menu">
+        </div>
+    </header>
+    <section class="headline">
+        <h2>I’m Mohamed Glad to see you!</h2>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="letsconnect">LET’S CONNECT</p>
+        <img src="src/img/social-media-icons.png" class="social-media-icons" alt="Social Media">
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link `(a tag)`